### ISSUE TITLE
Basic Delete/Edit Functionality

### ISSUE THREAD

**Issue by shadiaali** [Chatter-Laravel/core#13]
Could we get basic delete functionality for user's posts? A user can post but it is not possible to delete or edit your post.

**Comment by sbarbat** [comment]
That's true. But we will need to have some sort of historical data the editions/deletes.. Would you like to start with the development? 👍
